web_confirm_window_close/static/src/js/web_confirm_window_close.js

```diff
@@ -10,6 +10,7 @@ odoo.define('web_confirm_window_close', function (require) {
             if ($('html').find('.oe_form_dirty').length) {
                 e.preventDefault();
             }
+            return 'Your have unsaved data. Are you sure you want to close window?';
```

Comment:
As discussed earlier and as @Tardo found here

https://developer.mozilla.org/en-US/docs/Web/API/WindowEventHandlers/onbeforeunload#Notes
> Starting with Firefox 44, Chrome 51, Opera 38 and Safari 9.1, a generic string not under the control of the webpage will be shown instead of the returned string.


Any string returned would be ignored anyways and in older browser string would work just find and display their default generic dialog so better way is it to return blank string.

Please return blank string instead of this custom message